README.md

```diff
@@ -42,17 +42,58 @@ data:
 
 ## Usage
 
-### Environment Variables
-The plugin requires a certain set of Environment Variables to be able to run correctly.
+### Configuration
+The plugin requires some configuration to connect to Vault. The parameters are:
 
-| Name            | Description                | Supported            |
+| Name            | Description                | Notes                |
 | --------------- | -------------------------- | -------------------- |
-| AVP_VAULT_ADDR  | Address of your Vault      | N/A                  |
-| AVP_PATH_PREFIX | Prefix of the vault path to look for the secrets | N/A       |
-| AVP_TYPE        | The type of Vault backend  | `vault` and `secretmanager` |
-| AVP_AUTH_TYPE   | The type of authentication | vault: `approle, github`   secretmanager: `iam` |
+| VAULT_ADDR     | Address of your Vault      | N/A                  |
```

Comment:
These still have to have the prefix, right?

Reply:
Yeah only when specified as env variables

Reply:
Oh so its not needed when a config or secret, got it